packages/react/package.json

```diff
@@ -49,7 +49,7 @@
     "@lens-protocol/domain": "workspace:*",
     "@lens-protocol/shared-kernel": "workspace:*",
     "@lens-protocol/storage": "workspace:*",
-    "graphql": "15.5.1",
```

Comment:
Does anyone remember why it was using strict versioning?

Reply:
No reason I can remember from my side